blueocean-events/src/main/java/io/jenkins/blueocean/events/BlueMessageEnricher.java

```diff
@@ -64,7 +66,15 @@
 
     @Override
     public void enrich(@Nonnull Message message) {
+        try {
+            maybeEnrichMessage(message);
+        } catch (Exception e) {
```

Comment:
This looks fine. There is one downside though, masking exceptions could have front end side effects as many things are tied to sse events. If they suddenly stops appearing then only way to find will be to look in log for these warning messages.

Maybe we can do something like this:

```
// Add BlueEventProps.blueocean_unexpectederror enum up in this file
jobChannelMessage.set(BlueEventProps.blueocean_unexpectederror, e.getMessage());
```

Even if frontend doesn't process such error message but atleast will be easier to debug or find cause of UI not behaving as expected?

Reply:
possibly - risk of flooding is a bit high though.... but like you said, it is probably broken a bit.

Reply:
I think if things aren't working though, we tend to ask for a support bundle (easier to get than browser logs) so would tend to see a flood of this anyway so 🤷‍♀️